brave/brave-core
# New Onboarding WebUI

Resolves https://github.com/brave/brave-browser/issues/26378

~DO NOT MERGE BEFORE: https://github.com/brave/brave-core/pull/15674~

The PR adds the new onboarding UI. It replaces the previous `brave://welcome` UI.

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
Visit `brave://welcome` and go through the steps to ensure all functionality is working

1. Ensure single import is working
2. Ensure bulk import (multiple profiles) is working
3. Ensure importing from Safari is working

Background animations only work if hardware acceleration is enabled and "Reduce motion" flag disabled. The is to avoid animating on the CPU and respect accessibility. We need to ensure animations are not playing in the following scenarios:

1. Disable HW accel `--disable-gpu`
2. Enable HW accel and Reduced motion should be checked
